velox/exec/Driver.h

```diff
@@ -244,6 +244,36 @@ struct DriverCtx {
   std::optional<common::SpillConfig> makeSpillConfig(int32_t operatorId) const;
 };
 
+/// Enum for operator methods we want to track.
+enum class OpMethod {
```

Comment:
naming: do not abbreviate: OperatorMethod.

Reply:
I can change that, although I don't see how Op is not enough in this case.